packages/react-core/src/layouts/Grid/GridItem.js

```diff
@@ -37,7 +37,7 @@ const propTypes = {
 const defaultProps = {
   children: null,
   className: '',
-  span: null,
+  span: 1,
```

Comment:
Is this currently spanning 12? If that is the case that is actually the default preferred by core: https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/layouts/Grid/docs/code.md#usage (see `.pf-m-{1-12}-col{-on-[breakpoint]}` row)

Reply:
Yeah it spans over 12. Wow, didn't saw that. Okay we can close this one based on this, but it feels really odd to set span for each grid item.